style.css

```diff
@@ -1,6 +1,6 @@
 body {
   color: brown;
-  background-color: white;
```

Comment:
background color is repeated twice it would  be better if u could remove one of them although they have no effect on the style.